.gitignore

```diff
@@ -5,6 +5,9 @@
 *.so
 *.dylib
 
+# Output path for local repoman generation
```

Comment:
Why move abbreviations to /bicep folder?  We'll want to use those for TF as well.

Reply:
TF can directly use the [aztfmod/azurecaf](https://github.com/aztfmod/terraform-provider-azurecaf/tree/main/azurecaf)  provider  resources to directly generate resource names with caf abbreviation in the terraform module code. [here is how it is done for the applicationinsights.tf ](https://github.com/kaizentm/azure-dev/blob/cse/add-terrafrom-modules/templates/common/infra/terraform/applicationinsights.tf#L16:~:text=resource%20%22azurecaf_name,applicationinsights_name.result)

Reply:
Can you give an example of what the names would look like? Are they the same as bicep?

Reply:
ye names will match. I give you access to a deployed env on azure using terraform as a sample